common/src/main/java/com/microsoft/identity/common/internal/telemetry/TelemetryEventStrings.java

```diff
@@ -192,6 +192,8 @@
         public static final String LOCAL_ACQUIRE_TOKEN_SILENT= "103";
         public static final String LOCAL_GET_ACCOUNTS= "106";
         public static final String LOCAL_REMOVE_ACCOUNT= "107";
+        public static final String LOCAL_DEVICE_CODE_FLOW_ACQUIRE_URL_AND_CODE = "108";
```

Comment:
These constants look to be out of range for the DCF IDS (600-699)

For this, we'll need to update the value to be in this range, and update the spec to account for which value that you use:
- [Spec -- Msft corpnet reqd](https://microsoft-my.sharepoint-df.com/:x:/p/sagonzal/EXSrr4vM1utAqQQfD6bMln4BYKwjrqh3cagiNJWPVNjLzw?e=DH7Kmd)

Reply:
I think that excel sheet might only apply to the codes in `PublicApiId`? I think so, because the codes that are already in `TelemetryEventStrings` don't seem to match up with what's in the excel sheet.

Reply:
You are right; I am confused by who/what is consuming _these values_. Let's get some input from @shahzaibj (and maybe @sangonzal or @logdog16 if they know who or what is supposed to consume these values: these numbers don't match what's documented in the API ID Excel Sheet)

Reply:
These constants were added as part of this PR:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/519

Haven't located a spec capturing the numbers (yet)

Reply:
Gotcha, let me know if I need to make any changes

Reply:
Let's get input from @shahzaibj on Monday; I can't locate a spec that captures what these values are or should be

Reply:
@iambmelt Thanks for catching this. The values declared in the excel sheet you shared are the Api Ids for the Public APIs in MSAL. These are declared in `PublicApiId.java` in our code base, 

However, the values declared in this file i.e `TelemetryEventStrings.java` actually seem to capture an API ID for the internal controller methods in MSAL/common/broker. So there is an API Id for the `acquireTokenSilent` method in `LocalMsalController` vs one for the similar method in `BrokerMsalController` -- so these are different from the Public Api. 

With that said, even I'm not aware if there was ever a spec for these and/or if these were documented anywhere. Perhaps since these are internal, we could use anything for them? TBH, I'm not so sure. I'll see if I can track down any document that captures these, but for the time being @t-fadura let's make sure that the values we put here are different from the ones we put in `PublicApiId.java` as these fundamentally represent a different thing. I'm not too worried about the actual values you put here, let's just put anything but just make sure it is not same as public api id.